include/openPMD/IO/ADIOS/ADIOS1IOHandlerImpl.hpp

```diff
@@ -75,7 +75,7 @@ namespace openPMD
         void listAttributes(Writable*, Parameter< Operation::LIST_ATTS > &) override;
 
         virtual int64_t open_write(Writable *);
-        virtual ADIOS_FILE* open_read(std::string const& name);
+        virtual ADIOS_FILE* open_read(std::string name);
```

Comment:
not sure if it's clever to change this as well

Reply:
Internally, this just passes the ```.c_str()``` into ```adios_read_open_file```. There won't be any benefit from moving or creating a temporary as the string is not sunk. If anything, this could be changed to
```cpp
virtual ADIOS_FILE* open_read(char * const name);
```